{{cookiecutter.project_slug}}/requirements/requirements.in

```diff
@@ -0,0 +1,8 @@
+# Flask and related plugins
+flask
+flask-cors
+flask-apispec
+
+# Other common dependencies
+raven[flask]
+python-jose
```

Comment:
Aren't these in the base image now? You do need to specify here

```
-r /opt/base-requirements.txt
```

Reply:
It's already added [here](https://github.com/DD-DeCaF/cookiecutter-flask-microservice/pull/97/files#diff-21cd6e8e5fd83b2e927adbe64c6966e6R26). Or maybe you meant it's preferable to include it in `requirements.in`?

Reply:
Two things:

1. Since those dependencies are already specified and installed in the base image, they can be removed here.
2. `-r /opt/base-requirements.txt` (or `-r /opt/sql-requirements.txt`) needs to be included such that all the different version constraints are still satisfied.

Reply:
> 1. Since those dependencies are already specified and installed in the base image, they can be removed here.

Ah right. I was thinking it's nice to explicitly describe these requirements, since they're essential for the services. But I guess there's little point in that when it's easy to look up the base requirements anyway, and it adds a lot of duplication. I'll remove them then.

> 2. `-r /opt/base-requirements.txt` (or `-r /opt/sql-requirements.txt`) needs to be included such that all the different version constraints are still satisfied.

Yeah. It was previously just part of the command in the Makefile, but it's explicitly included in `requirements.in` now.